cljfmt/src/cljfmt/core.cljc

```diff
@@ -1,24 +1,25 @@
 (ns cljfmt.core
   #?@(:clj
-       [(:require
-         [clojure.java.io :as io]
-         [clojure.zip :as zip]
-         [rewrite-clj.node :as n]
-         [rewrite-clj.parser :as p]
-         [rewrite-clj.zip :as z
-          :refer [append-space edn skip whitespace-or-comment?]])
-        (:import java.util.regex.Pattern)]
-       :cljs
-       [(:require
-         [cljs.reader :as reader]
-         [clojure.zip :as zip]
-         [rewrite-clj.node :as n]
-         [rewrite-clj.parser :as p]
-         [rewrite-clj.zip :as z]
-         [rewrite-clj.zip.base :as zb :refer [edn]]
-         [rewrite-clj.zip.whitespace :as zw
-          :refer [append-space skip whitespace-or-comment?]])
-        (:require-macros [cljfmt.core :refer [read-resource]])]))
+      [(:require
+        [clojure.java.io :as io]
+        [clojure.zip :as zip]
+        [rewrite-clj.node :as n]
+        [rewrite-clj.parser :as p]
+        [rewrite-clj.zip :as z
+         :refer [append-space edn skip whitespace-or-comment?]])
+       (:import java.util.regex.Pattern)]
+      :cljs
+      [(:require
+        [cljs.reader :as reader]
+        [clojure.zip :as zip]
+        [clojure.string :as str]
+        [rewrite-clj.node :as n]
+        [rewrite-clj.parser :as p]
+        [rewrite-clj.zip :as z]
+        [rewrite-clj.zip.base :as zb :refer [edn]]
+        [rewrite-clj.zip.whitespace :as zw
+         :refer [append-space skip whitespace-or-comment?]])
+       (:require-macros [cljfmt.core :refer [read-resource]])]))
```

Comment:
It looks like these lines are just correcting spacing. While that's good, could you put that into a different commit, or different PR? It makes it hard to tell whether the namespace declaration has changed or not.